pkgs/by-name/li/libiberty/package.nix

```diff
@@ -14,6 +14,12 @@ stdenv.mkDerivation {
 
   postUnpack = "sourceRoot=\${sourceRoot}/libiberty";
 
+  # needed until config scripts are updated to not use /usr/bin/uname on FreeBSD native
+  # updateAutotoolsGnuConfigScriptsHook doesn't seem to work here
```

Comment:
updateAutotoolsGnuConfigScriptsHook only looks in `.`, so instead of using `sourceRoot` to change directory after `unpackPhase` we would need to `cd` in `preConfigure` to make it work…

Alternatively we could make the directory configurable in the setup hook 🤔